com.unity.render-pipelines.universal/Editor/ShaderGUI/ShaderGraphLitGUI.cs

```diff
@@ -33,7 +33,8 @@ public static void UpdateMaterial(Material material, MaterialUpdateType updateTy
             if (updateType == MaterialUpdateType.CreatedNewMaterial)
                 material.globalIlluminationFlags = MaterialGlobalIlluminationFlags.BakedEmissive;
 
-            BaseShaderGUI.UpdateMaterialSurfaceOptions(material, automaticRenderQueue: false);
+            bool automaticRenderQueue = GetAutomaticQueueControlSetting(material);
+            BaseShaderGUI.UpdateMaterialSurfaceOptions(material, automaticRenderQueue);
```

Comment:
Looking at this again, it looks like the render queue doesn't get properly updated with the queue offset when in manual mode. I think this might be tricky too as setting the renderQueue needs to include the offset, but then you'd update the new render queue again with the offset.

Reply:
QueueOffset happens in BaseShaderGUI.SetupMaterialBlendModeInternal, and it is _always_ applied as long as the property exists (called from BaseShaderGUI.UpdateMaterialSurfaceOptions).  This PR doesn't change that.

Reply:
It's not applied when QueueControl is set to Auto as shown here:
![AutoQueueControlSortingPriority](https://user-images.githubusercontent.com/76977132/121597181-4105c600-c9f5-11eb-938a-0f2e47adcb31.gif)
If this is intended I think the sorting priority should be hidden as this is a bit confusing.